graylog2-web-interface/src/components/search/MessageFieldDescription.jsx

```diff
@@ -54,6 +55,13 @@ const MessageFieldDescription = React.createClass({
 
     return termsMarkup;
   },
+  _getFieldValue() {
+    if (this.props.linkedFields.includes(this.props.fieldName)) {
+      return <a href={this.props.fieldName}>{this.props.fieldName}</a>
```

Comment:
Ah, now you're missing the highlight in the content of the `<a>` tag. 😉

```javascript
return <a href={this.props.fieldName}>{this.props.possiblyHighlight(this.props.fieldName)}</a>
```

Reply:
Fixed this!